test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/ProjectRestoreInfoBuilder.cs

```diff
@@ -306,12 +307,14 @@ private static IVsReferenceItem ToToolReference(LibraryRange libraryRange)
             return new VsReferenceItem(libraryRange.Name, properties);
         }
 
-        private static IVsReferenceItem ToPackageDownload(DownloadDependency library)
+        private static IVsReferenceItem ToPackageDownload(IGrouping<string, DownloadDependency> library)
```

Comment:
A simple test with a packageDownload with multiple versions would be great.

Reply:
There's a test. The test infrastructure was bad.

https://github.com/NuGet/NuGet.Client/blob/a114e45c532a058746a6c31fc1a65f496d73e4fa/test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs#L1406

Reply:
Added this detail in the PR description to help other reviewers.